migrate/live-sync.md

```diff
@@ -0,0 +1,282 @@
+---
+title: Live sync
+excerpt: Synchronize updates to a primary postgres database instance to Timescale Cloud service in real-time
+products: [cloud]
+keywords: [migration, low-downtime, backup]
+tags: [recovery, logical backup, replication]
+---
+
+import MigrationPrerequisites from "versionContent/_partials/_migrate_prerequisites.mdx";
+import SetupConnectionStrings from "versionContent/_partials/_migrate_live_setup_connection_strings.mdx";
+
+
+# Live sync
+
+You use the live-sync Docker image to synchronize changes in real-time from a PostgreSQL database 
+instance to a $SERVICE_LONG. You run live-sync continuously, turning PostgreSQL into a primary database 
+with a $SERVICE_LONG as a logical replica. This enables you to leverage $CLOUD_LONG’s real-time analytics 
```

Comment:
Can we also add a mention that user gets to to choose to replicate all or some of their data with choosen tables.

Reply:
@VineethReddy02 @billy-the-fish We should also highlight the fact that this is not intended for migration, but just to keep synching data for analytics. We expect the primary to be running all the time.

Though it could be a potential migration path, the UX is not ready for that yet.

Reply:
Done